content/_data/upgrades/2-249-1.adoc

```diff
@@ -0,0 +1,70 @@
+==== User interface improvements
+
+The Jenkins user interface includes significant user interface improvements.
+The improvements provide theming support and a more modern, attractive user interface.
+
+Some plugins may need additional changes to adapt to the latest user interface changes.
+
+==== Microsoft .NET Framework 2.0 support removed
+
+image::/images/post-images/2020/07-windows-support-updates/winsw-logo.png[title="WinSW Logo", float=right, width="224px"]
+
+Starting from this release, we won't support .NET Framework 2.0 for launching the Jenkins controller or agents as Windows services.
+Microsoft.NET Framework 4.0 or above is now required for using the default service management features.
+
+This release also upgrades link:https://github.com/winsw/winsw[Windows Service Wrapper (WinSW)] from 2.3.0 to 2.9.0 and replaces the bundled binary from .NET Framework 2.0 to 4.0.
+There are many improvements and fixes in these versions,
+big thanks to link:https://github.com/NextTurn[NextTurn] and all other contributors.
+You can find the full WinSW changelog link:https://github.com/winsw/winsw/releases[here],
+just a few highlights important to Jenkins users:
+
+* Prompt for permission elevation when administrative access is required.
+  Now Jenkins users do not need to run the agent process as Administrator to install the agent as a service from GUI.
+* Enable TLS 1.1/1.2 in .NET Framework 4.0 packages on Windows 7 and Windows Server 2008 R2.
+* Enable strong cryptography when running .NET Framework 4.0 binaries on .NET 4.6.
+* Support security descriptor string in the Windows service definition.
+* Support 'If-Modified-Since' and proxy settings for automatic downloads.
+* Fix Runaway Process Killer extension so that it does not kill wrong processes with the same PID on startup.
+* Fix the default domain name in the `serviceaccount` parameter (jira:JENKINS-12660[])
+* Fix archiving of old logs in the `roll-by-size-time` mode.
+
+As you may see, there are many improvements available with this version,
+and we hope that it will make Windows service installation even more reliable.
+Some of the changes in WinSW also replaced old workarounds in the Jenkins core,
+making the code more maintainable.
+
+===== Use-cases affected by .NET Framework 2.0 support removal
+
+If you use .NET Framework 2.0 to run the Jenkins Windows services,
+the following use-cases are likely to be affected:
+
+* Installing the Jenkins controller as a Windows service from Web UI.
+  The official MSI Installer supports .NET Framework 2.0 for the moment, but it will be changed in future versions.
+* Installing agents as Windows services from GUI.
```

Comment:
@slide can you confirm that this statement is still accurate?  MSI installer has not yet dropped support for .NET framework 2.0?

Reply:
Yes, it looks like the .exe.config file still has 2.0 support. 

https://github.com/jenkinsci/packaging/blob/INFRA-910-core-release-automation/msi/build/jenkins.exe.config#L7